theme/less/components/toast.less

```diff
@@ -129,5 +129,11 @@
     padding: .5em 2em;
     font-family: Avenir, Helvetica, Arial, sans-serif;
     animation-duration: .15s;
-    display: grid;
-}
\ No newline at end of file
+    display: flex;
```

Comment:
I think we have some issue with hover. See:
![image](https://user-images.githubusercontent.com/28541902/173387905-ce434fc3-9912-4090-8425-4af601b91876.png)

Reply:
Done ! This now extends the `.fr-btn` and use DSFR colors